bitmap-plasma/app/src/main/jni/plasma.c

```diff
@@ -140,7 +140,7 @@ static uint16_t  palette[PALETTE_SIZE];
 static uint16_t  make565(int red, int green, int blue)
 {
     return (uint16_t)( ((red   << 8) & 0xf800) |
-                       ((green << 2) & 0x03e0) |
+                       ((green << 3) & 0x07e0) |
                        ((blue  >> 3) & 0x001f) );
 }
```

Comment:
that is very one.  The change is piggyback on this app's c original coding style.  probably at later time we got to make them to be more conformance with the latest coding style for all samples in this repo.

thank you